src/localization/locales/en.json

```diff
@@ -79,7 +79,7 @@
                 "local": {
                     "labelInElectron":"Local folder",
                     "labelInBrowser":"Local folder (This is only enabled in the desktop version from: https://github.com/Azure/azure-iot-explorer/releases)",
-                    "infoText": "Use your local folder as a model repository. The interface files you wish to use should have a json extension and a file name equal to the Interface ID with all colons (':') removed. For example: urn:contoso:com:EnvironmentalSensor:1 should be named as urncontosocomEnvironmentalSensor1.json",
+                    "infoText": "Use your local folder as a model repository. The model definition files you wish to use should have a json extension. We will retrieve model definition by dtmi through searching the matching '@id' from the files, and the first matching file content would be retrieved.",
```

Comment:
Consider avoiding acronyms here. Also I'm a little confused by the wording. Does this mean that it will return the first interface definition where the IDs match?

Reply:
Yes. Added the full name for dtmi, and @id is what is defined in dtdl 
![image](https://user-images.githubusercontent.com/5489222/78705526-74fdcc80-78c2-11ea-8071-e8ad25b692ae.png)
I am kinda struggling to find the right wording for this.

Reply:
Yeah I like that! "The first JSON file with an `@id` property matching the digital twin model identifier (DTMI) will be retrieved."